### DISCUSSION TITLE
Edit number of ticks or tick location on bounds/axis

### DISCUSSION THREAD

**Discussion by kaathaarinaa** [pyvista/pyvista#3921] [Q&A]
Hi,

I would like to edit the number of ticks that appear on the ```z``` bounds/axis. I created a list with the tick locations but do not know how to use them in the `pv.Plotter()`. In `pv.Charts2D()` the tick locations can be edited using `chart.y_axis.tick_locations = (0.2, 0.4, 0.6, 1, 1.5, 2, 3)`. is there a way of doing that for the below code?
```
import pyvista
from pyvista import examples

mesh = examples.load_uniform()  # Get dummy data
# Plot
pl = pyvista.Plotter()
actor = pl.add_mesh(mesh, color='red')
z_ticks_location = [0, 10, 20, 30]
pl.show_bounds(bounds=[-25, 90, -30, 30, 0, 35], xlabel='x', ylabel='y', zlabel='z', grid=True, location='outer')
pl.show()
```

**Comment by banesullivan** [comment]
See https://github.com/pyvista/pyvista/pull/3977
